packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+## Anchor Transaction
+
+The first transaction in every Taiko L2 block to perform data validation and enable L1-to-L2 communication.
+
+## Bridge
+
+The Taiko bridge allowing users to bridge Ether and custom ERC-20 tokens from and to Taiko.
+
+## Commit Hash
+
+The Keccak-256 hash of a proposed block’s beneficiary and transaction list.
+
+## Cross-chain communication (messaging)
+
+Bridging over messages using a pair of smart contracts that use state proofs to verify data across chains.
```

Comment:
```suggestion
Sending secure messages across two chains by presenting a merkle proof of the source chain message on the destination chain. This is enabled because Taiko blocks are equivalent to Ethereum blocks.
```